e2e/editPage.spec.ts

```diff
@@ -0,0 +1,374 @@
+
+    const TEST_UNLINKED_PAGE_TITLE = "Test Unlinked Page"
+    const TEST_UNLINKED_PAGE_FILENAME = titleToPageFileName(
+      TEST_UNLINKED_PAGE_TITLE
+    )
+    const TEST_PAGE_TITLE_ENCODED = encodeURIComponent(
+      TEST_UNLINKED_PAGE_FILENAME
+    )
+
+    const DEFAULT_IMAGE_TITLE = "isomer-logo.svg"
+    const ADDED_IMAGE_TITLE = "balloon"
+    const ADDED_IMAGE_PATH = "images/balloon.png"
+
+    const ADDED_FILE_TITLE = "singapore-pages"
+    const ADDED_FILE_PATH = "files/singapore.pdf"
+
+    const LINK_TITLE = "link"
+    const LINK_URL = "https://www.google.com"
+
+    const UNLINKED_PAGE = {
+      frontMatter: {
+        title: TEST_UNLINKED_PAGE_TITLE,
+        permalink: "/permalink",
+        variant: "tiptap",
+        description: "",
+      },
+    }
+
+    const BASE_SETTINGS = {
+      title: "Title",
+      description: "An Isomer site of the Singapore Government",
+      favicon: "/images/favicon-isomer.ico",
+      shareicon: "/images/isomer-logo.svg",
+      google_analytics_ga4: "",
+      is_government: false,
+      contact_us: "/contact-us/",
+      feedback: "",
+      faq: "/faq/",
+      show_reach: true,
+      logo: "/images/isomer-logo.svg",
+      url: BASE_SEO_LINK,
+      social_media: {
+        facebook: "https://www.facebook.com/YourFBPage",
+        linkedin: "https://www.linkedin.com/company/YourAgency",
+        twitter: "https://www.twitter.com/YourTwitter",
+        youtube: "https://www.youtube.com/YourYoutube",
+        instagram: "https://www.instagram.com/your.insta/",
+        telegram: "",
+        tiktok: "",
+      },
+      colors: {
+        "primary-color": "#ff0000",
+        "secondary-color": "#ff0000",
+        "media-colors": [
+          { title: "media-color-one", color: "#ff0000" },
+          { title: "media-color-two", color: "#ff0000" },
+          { title: "media-color-three", color: "#ff0000" },
+          { title: "media-color-four", color: "#ff0000" },
+          { title: "media-color-five", color: "#ff0000" },
+        ],
+      },
+      "facebook-pixel": "",
+      "linkedin-insights": "",
+    }
+
+    test.beforeEach(async ({ page, context }) => {
+      setEmailSessionDefaults(context, "Email admin")
+      const api = await getApi(await context.storageState())
+      await api.post("v2/sites/e2e-email-test-repo/pages/pages", {
+        data: {
+          content: UNLINKED_PAGE,
+          newFileName: TEST_UNLINKED_PAGE_FILENAME,
+        },
+      })
+      await page.goto(
+        `/sites/${E2E_EMAIL_TEST_SITE.repo}/editPage/${TEST_PAGE_TITLE_ENCODED}`
+      )
+    })
+
+    test("Edit page (unlinked) should have correct colour", async ({
+      page,
+      context,
+    }) => {
+      // Arrange
+      const api = await getApi(await context.storageState())
+      const header = page.locator("#display-header")
+
+      // Act
+      await api.post("v2/sites/e2e-email-test-repo/settings", {
+        data: BASE_SETTINGS,
+      })
```

Comment:
question: does the colour get updated immediately without refreshing the page?

Actually I'm a bit confused here. We seem to be setting the colour then testing if the page uses the same colour (testing 2 things), rather than testing if the page uses the colour that was originally set on the settings page (testing 1 thing).

Reply:
Hmm i feel that this is actually testing a single thing, that setting a colour causes the page colour to change as well.

I think we can use your suggestion (check colour of edit page vs colour specified on settings) if we change the default colour of the repo though! Otherwise we have a fallback colour - we need to be able to distinguish between using the fallback colour and the actual colour

Reply:
i actually didn't realise this - the test still passes but i agree w/ the reload

Reply:
added a `page.reload`